clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -343,6 +343,10 @@ Changes in existing checks
   <clang-tidy/checks/readability/static-accessed-through-instance>` check to
   identify calls to static member functions with out-of-class inline definitions.
 
+- Improved :doc:`modernize-avoid-bind
```

Comment:
put this in alphabetical order, use double `` for return, as this is C++ keyword, also same for ``void``